ProgramFlow/ProgramFlow/Flow.cs

```diff
@@ -40,8 +55,26 @@ public static DriverAction DriveSafelyIfElse(LightColor color)
         /// <returns>The correct <see cref="DriverAction"/> value.</returns>
         public static DriverAction DriveSafelySwitch(LightColor color)
         {
+
             // TODO: Fix this method so it passes all associated tests. A switch statement must be used to return the correct values.
-            return (DriverAction)24;
+            switch (color)
+            {
+                case LightColor.Green:
+                    return (DriverAction.ProceedWithCaution);
+                    break;
```

Comment:
The break statements after return keywords are unnecessary and generate compiler warnings because they are unreachable.